### ISSUE TITLE
It does not escape special Xml characters(such as ", $) when calling toXml function

### ISSUE THREAD

**Issue by RickyJiao** [buglabs/node-xml2json#75]
Please check below code snippet:
```
  var xml2Json = require("xml2json")
  var test = {text:["\"&aa","\"bb"]}
  console.log(xml2Json.toXml(test))
```
it will print:
```
' text=""&aa" text=""bb">'
```

**Comment by c4milo** [comment]
@RickyJiao, I just published version 0.6.0 which would sanitize properly those values. To use it, make sure you pass an options object to `toXml` like so:

```javascript
var xml2Json = require("xml2json")
var test = {text:["\"&aa","\"bb"]}
var result = xml2Json.toXml(test, { sanitize: true })
console.log(result)
```

**Comment by c4milo** [comment]
This should be fixed, I would re-open if you see it happening in your side.
